### ISSUE TITLE
Misplaced opening brace

### ISSUE THREAD

**Issue by Zorgatone** [palantir/tslint#935]
The following snippets gives me the "Misplaced opening brace" error on tslint version "3.2.2-dev.1".

Is it supposed to be like that? And how do I need to reformat the code to make it happy?

```TypeScript
let geoConfig: {
    maximumAge?: number;
    timeout?: number;
    enableHighAccuracy?: boolean;
} = {
    maximumAge: 3000,
    timeout: 5000,
    enableHighAccuracy: false // true might cause some errors
};
```

**Comment by JKillian** [comment]
Hey @Zorgatone, thanks for noticing this, seems like a bug. I've filed #937 to fix this. For now, I'd just surround your code with disable/enable flags:

```ts
/* tslint:disable:one-line */
...variable declaration here...
/* tslint:enable:one-line */
```

**Comment by JKillian** [comment]
The fix for this has merged and should be available in the next release of TSLint (likely about a week).

**Comment by Zorgatone** [comment]
Wow that was quick!

Thank for the fix

**Comment by adidahiya** [comment]
Fix is now available in `v3.4.0-dev.1`
